File: Interfaz/ProyectoFinal/src/app/Parametros/Entrada/ObtenerEstadisticasIn.ts
export class ObtenerEstadisticasIn{
    
    private enumeradoGrafica!: number;
    private fechaDesde!: Date;
    private fechaHasta!: Date;

    constructor(){}

    public get FechaDesde(): Date {
        return this.fechaDesde;
    }
    public set FechaDesde(value: Date) {
        this.fechaDesde = value;
    }
    public get FechaHasta(): Date {
        return this.fechaHasta;
    }
    public set FechaHasta(value: Date) {
        this.fechaHasta = value;
    }
    public get EnumeradoGrafica(): number {
        return this.enumeradoGrafica;
    }
    public set EnumeradoGrafica(value: number) {
        this.enumeradoGrafica = value;
    }
}

File: ProyectoFinal/ProyectoFinal/Parametros/Entrada/ObtenerEstadisticasIn.cs
using ProyectoFinal._2_Dominio.Entidades;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Entrada
{
    public class ObtenerEstadisticasIn
    {
        public EnumeradoGraficas EnumeradoGrafica { get; set; }
        public DateTime FechaDesde { get; set; }
        public DateTime FechaHasta { get; set; }
    }
}